src/MainMenu/MenuItem/MenuItem.tsx

```diff
@@ -46,6 +46,7 @@ export const MenuItem = ({
         onClick={onNavigate}
         disabled={disabled}
         selected={selected}
+        data-umami-event={id}
```

Comment:
I think here I would prefer to get `umamiEvent` from the props. Sidebar decides what should be the event shape.